src/JsRoot/GraphData.tsx

```diff
@@ -38,14 +39,19 @@ export interface Page0D extends IPage {
 	dimensions: 0;
 }
 
+export interface PageCT extends IPage {
+	dimensions: 'CT';
+	resultsUrl: string;
+}
```

Comment:
What does CT stand for? Is this data format dictated by the backend response? We could use some renames to clarify that Page types are distinguishable by dimension count or this 'CT' value.

Reply:
I'm not sure but I was to use name 'CT' when referring to patient objects like meshes etc.

Reply:
this is also puzzling me - what is stored in the dimesions ? At first glance this looks like: X,Y,Z or cm or mm.

Reply:
It has int value representing number of dimensions for input. So usually it's 0, 1, 2 or 3.